libs/ballot-interpreter/src/hmpb-rust/debug.rs

```diff
@@ -27,7 +33,7 @@ pub fn draw_qr_code_debug_image_mut(
     detection_areas: &[Rect],
 ) {
     for detection_area in detection_areas {
-        draw_hollow_rect_mut(canvas, (*detection_area).into(), ORANGE);
```

Comment:
I'm assuming you're referring to this:
> One should avoid implementing `Into` and implement `From` instead. Implementing `From` automatically provides one with an implementation of `Into` thanks to the blanket implementation in the standard library.

That means that you should avoid `impl Into<A> for B` and instead do `impl From<B> for A` instead. Then you can use `Into::into` because of this stdlib implementation:

```rust
// From implies Into
#[stable(feature = "rust1", since = "1.0.0")]
impl<T, U> Into<U> for T
where
    U: From<T>,
{
    /// Calls `U::from(self)`.
    ///
    /// That is, this conversion is whatever the implementation of
    /// <code>[From]&lt;T&gt; for U</code> chooses to do.
    #[inline]
    fn into(self) -> U {
        U::from(self)
    }
}
```

The above automatically implements `Into<U> for T` whenever `U` has a `From<T>` implementation.

The reason I could no longer use `.into()` here is because of the so-called ["Orphan Rule"](https://github.com/Ixrec/rust-orphan-rules?tab=readme-ov-file#what-are-the-orphan-rules), which requires that we cannot implement e.g. `From` between two types if our crate doesn't define either of them. Previously we had defined `Rect` in `ballot-interpreter` and had a `impl From<Rect> for imageproc::Rect::Rect` [here](https://github.com/votingworks/vxsuite/pull/4432/files#diff-eae8c1e5358ad1d7777f1d998ed1c7f6c3ec9e820c763b3da27fb282e704c8beL210-L214) which allowed us to use `Into::into` to convert between the two. Now that `Rect` is defined in `types-rs`, the Orphan Rule says we cannot define `From` between it and `imageproc::rect::Rect` since we don't define either of those types. The typical workaround for this is either to define a wrapper type to convert through (since we do own that one) or to do what I did and just write a normal function to be called manually.